### PR TITLE
Add org.opencontainers.image.* labels

### PR DESCRIPTION
**Pull request by williamdes**
Related to https://github.com/phpmyadmin/docker/pull/274 and https://github.com/docker-library/official-images/pull/7288

Spec: https://github.com/opencontainers/image-spec/blob/v1.0.1/annotations.md

### CODE REVIEW

**Comment by williamdes** [comment]
@J0WI can you let me know what you think about this PR ?

File: Dockerfile-debian.template [hunk 1]
```diff
@@ -64,6 +64,21 @@ RUN set -ex; \
...
+ARG BUILD_DATE
+ARG VCS_REF
+
+LABEL maintainer="The phpMyAdmin Team <developers@phpmyadmin.net>" \
+    org.opencontainers.image.title="Official phpMyAdmin Docker image" \
+    org.opencontainers.image.description="Run phpMyAdmin with Alpine, Apache and PHP FPM." \
+    org.opencontainers.image.authors="The phpMyAdmin Team <developers@phpmyadmin.net>" \
+    org.opencontainers.image.vendor="phpMyAdmin" \
+    org.opencontainers.image.documentation="https://github.com/phpmyadmin/docker#readme" \
+    org.opencontainers.image.licenses="GPL-2.0" \
```
Comment (J0WI) [inline Dockerfile-debian.template]: https://github.com/phpmyadmin/docker/blob/master/LICENSE is 3.0+, isn't it?

Comment (williamdes) [reply Dockerfile-debian.template]: "org.opencontainers.image.licenses License(s) under which contained software is distributed"
I hesitated but phpMyAdmin is GPL-2.0

Should I change ?

https://github.com/opencontainers/image-spec/blob/v1.0.1/annotations.md

Comment (J0WI) [reply Dockerfile-debian.template]: Oh ok. But therefore the SPDX identifier is deprecated: https://spdx.org/licenses/

Comment (williamdes) [reply Dockerfile-debian.template]: https://github.com/phpmyadmin/phpmyadmin/blob/master/composer.json#L14

6b63869f1c33fbacdc2b74ea2c3fdb0d0daa04b3

Fixed, thank you !

**Comment by williamdes** [comment]
> This is fine for me, even though I don't personally see much use of LABELs in this way.

They can be used to generate badges and maybe some GUIs use them so I added them

**Pull request merged**